test/last-run.js

```diff
@@ -159,15 +164,9 @@ describe('lastRun', function() {
       lastRun.capture(test);
     }
 
-    if (/v0.10/.test(process.version)) {
-      expect(extensions).toThrow('Only extensible functions can be captured');
-      expect(seal).toThrow('Only extensible functions can be captured');
-      expect(freeze).toThrow('Only extensible functions can be captured');
-    } else {
-      expect(extensions).toNotThrow();
-      expect(seal).toNotThrow();
-      expect(freeze).toNotThrow();
-    }
+    expect(extensions).not.toThrow();
+    expect(seal).not.toThrow();
+    expect(freeze).not.toThrow();
```

Comment:
The description of the test is now wrong. I think this test can just be removed completely.

Reply:
Got it. I'll remove this test case.